doc/network.tex

```diff
@@ -1625,8 +1647,8 @@ \subsection{Fairness and Flow-Control in the Multiplexer}
 
 A concrete implementation of a multiplexer may use a variety of data structures and heuristics to
 yield the overall best efficiency.
-For example, although, the multiplexing protocol itself is agnostic to the underlying structure of
-the data, the multiplexer may try to avoid to split small mini protocol messages into two segments.
+For example, although the multiplexing protocol itself is agnostic to the underlying structure of
+the data, the multiplexer may try to avoid spliting small mini protocol messages into two segments.
```

Comment:
Make it clear that the two small messages MUST be from the same miniprotocol for them to be merged into the same segment.